automation/taskcluster/lib/util.py

```diff
@@ -0,0 +1,27 @@
+import json
+import re
+
+
+def convert_camel_case_into_kebab_case(string):
```

Comment:
Is this used anywhere?

Reply:
Yup. Here: https://github.com/mozilla-mobile/fenix/pull/1220/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R136.
Btw, this is unrelated to the Kebab you had yesterday 😁

Reply:
Oh, wow, lack of sleep is hitting me: I skipped right by `tasks.py` because github had folded it for me due to being a large diff haha :)